server/index.js

```diff
@@ -25,6 +29,18 @@ app.use(morgan('dev'));
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
+app.use(passport.initialize());
+app.use(session({
```

Comment:
why are you using sessions? since we are using tokens for authentication.
Please explain and revert back to the use of tokens alone.

Reply:
it was part of the codebase... not really making use of it at the moment.